lib/data/model/now_playing_movies_model.dart

```diff
@@ -0,0 +1,29 @@
+import 'movie_model.dart';
+
+class NowPlayingMoviesModel {
```

Comment:
You are creating 4 model that are exactly the same
`NowPlayingMoviesModel, PopularMoviesModel, TopRatedMoviesModel and UpcomingMoviesModel`
You need to have only 1 model that could be named like `ResponseModel` because is a response from the service

Reply:
done, and thanks for this! DRY, no? jeje